outlines/base.py

```diff
@@ -12,9 +11,6 @@
     _update_dim_sizes,
 )
 
-# Allow nested loops, useful to run in notebooks
-nest_asyncio.apply()
```

Comment:
This was necessary of people would run into issues when running the OpenAI models in notebooks. There may be a better way to handle this but haven't found it yet

Reply:
Removing this was the quickest way to fix the exception I reported above. If you look closely, there's a problem with patching vLLM's event loop

Reply:
brought it back and wrapped it in try-except.